.github/workflows/docker-build.yml

```diff
@@ -0,0 +1,16 @@
+name: Docker-build
+on: [pull_request]
+jobs:
+
+  build:
+    name: Docker build
+    runs-on: ubuntu-latest
+    env:
+      GO111MODULE: on
+      TAG: network-resources-injector:latest 
+    steps:
+    - name: Check out code into the Go module directory
+      uses: actions/checkout@v2
+
+    - name: Build Dockerfile
+      run: docker build -f Dockerfile -t ${TAG} .
```

Comment:
nit: I see there is a docker build-and-push github action, do you know if we can use it for build image only?

Reply:
> btw, will this workflow also aim to push the docker image to github container registry?

I added it to latest commit with build image only! Thanks for this! I also see there is support for multi arch docker builds. I attempted it but it [hung](https://github.com/martinkennelly/network-resources-injector/actions/runs/632419982).